### ISSUE TITLE
About CcGAN

### ISSUE THREAD

**Issue by Kanon5535** [UBCDingXin/improved_CcGAN#9]
Hello, I'm Shi from Tokyo.

I have created my program with reference to your program.
However, when I ran it, I got the error shown in the photos.
By the way, when I change batch_size_d to 8, my program run, but the d_loss are always 0.
I tried to solved it, but I had no idea what to do, so I hope you can help me.
I'm Chinese, and I can read and speak Chinese, if you feel easy to write in Chinese you can use it.

![スクリーンショット 2022-09-16 205240](https://user-images.githubusercontent.com/113690922/190634557-46811fb2-f3df-4d4a-8f5d-7d9b36bb524d.png)
![スクリーンショット 2022-09-16 205615](https://user-images.githubusercontent.com/113690922/190634572-f7ccd02b-4c21-4271-9bca-d14fa6fc822b.png)

**Comment by UBCDingXin** [comment]
Hi Shi,

Thanks for your interest in our work.

As for your problem, I suggest double-checking the dimension of the output of each block in the discriminator. It seems there is a dimension mismatch problem.

Hope it can help.

**Comment by Kanon5535** [comment]
Hi Ding,

Thanks for replying.
I made some changes to my program based on what you told me, and the problem has been resolved.
And now I want to generate images with specific labels, for example, only the chair with 55° angle. I tried to write a test　program to load the trained model and generate the images, but it seems to fail when adding the labels. I would like to ask, how can I add the specific labels right?
I apologize for the inconvenience, but I would appreciate your reply.
